libtbx_refresh.py

```diff
@@ -51,14 +71,15 @@ def _install_setup_readonly_fallback(package_name: str):
     # Get the actual environment being configured (NOT libtbx.env)
     env = _get_real_env_hack_hack_hack()
 
-    # Update the libtbx environment pythonpaths to point to the source
-    # location which now has an .egg-info folder; this will mean that
-    # the PYTHONPATH is written into the libtbx dispatchers
-    rel_path = libtbx.env.as_relocatable_path(import_path)
-    if rel_path not in env.pythonpath:
-        env.pythonpath.insert(0, rel_path)
+    # As of PEP 660, the package metadata (dist-info) goes in the install dir,
+    # not the source dir. Add this location to the python path too.
```

Comment:
```suggestion
    # not the source dir. Add this location to the python path.
```
Same as DIALS now you removed the last block